app/src/main/res/values/themes.xml

```diff
@@ -39,6 +40,11 @@
     <item name="fontFamilyEmphasized">@font/shadows_into_light</item>
   </style>
 
+  <style name="LondonThemeBpkStarRating" parent="">
+    <item name="starColor">?bpkGray100Color</item>
+    <item name="starFilledColor">?colorAccent</item>
```

Comment:
Can we change this to `?npkPrimaryColor`? We are not setting `colorAccent` on those  themes

Reply:
Done